libopenage/engine.cpp

```diff
@@ -47,10 +47,9 @@ Engine *Engine::instance = nullptr;
 void Engine::create(util::Dir *data_dir, const char *windowtitle) {
 	// only create the singleton instance if it was not created before..
 	if (Engine::instance == nullptr) {
-		// initialize the default actions
-		input::actions::initialize_actions();
 		// reset the pointer to the new engine
 		Engine::instance = new Engine(data_dir, windowtitle);
+		Engine::instance->get_input_manager().initialize(); // until non hardcoded keybind
```

Comment:
@TheJJ @ChipmunkV  The segfault problem was because the InputManager initialization was made before the Engine instance was finalized. Thanks for the help.